docs/proposals/edgepodgetandrestart.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Add pod restart and status query functions for the edge node
+authors:
+- "@luomengY"
+  approvers:
+  creation-date: 2024-04-15
+  last-updated: 2024-04-15
+  status: implementable
+---
+
+# Add pod restart and status query functions for the edge node 
+
+
+## Motivation 
+
+When the edge node is offline, it is not possible to query the status of the edge node pod and restart the pod of the edge node through kubectl in the cloud. This feature provides support for querying and restarting the pod status of the edge node.
+
+### Goals
+
+- By using the `keadm get pod [flags]` command on edge nodes, the status of pod can be queried.
+- By using the `keadm restart pod [flags]` command on edge nodes, pod can be restarted.
```

Comment:
My personal opinion. keadm get and restart belong to the same category of operations. I think it's more appropriate to unify them under keadm subcommand, like keadm xxx get/restart. Also it would make it easier for future expanision. 
cc @WillardHu @fisherxu @wbc6080

Reply:
I think it may not be very suitable to include extended functions in keadm in the future, such as describe/logs/exec. Future extensions may not only be for pod resources, such as device, etc. So can we try to maintain some command styles of kubectl in the upstream k8s as much as possible?